src/components/templates/order-table/use-order-column.tsx

```diff
@@ -64,59 +61,56 @@ const useOrderTableColums = () => {
               }}
               color={getColor(row.index)}
             />
-          </Table.Cell>
         ),
       },
       {
         Header: "Fulfillment",
         accessor: "fulfillment_status",
-        Cell: ({ cell: { value }, index }) => (
-          <Table.Cell key={index}>{value}</Table.Cell>
+        Cell: ({ cell: { value } }) => (
+          {value}
         ),
       },
       {
         Header: "Payment status",
         accessor: "payment_status",
-        Cell: ({ cell: { value }, index }) => (
-          <Table.Cell key={index}>{decideStatus(value)}</Table.Cell>
+        Cell: ({ cell: { value }  }) => (
+          {value}
         ),
       },
       {
         Header: "Sales Channel",
         accessor: "sales_channel",
-        Cell: ({ cell: { value }, index }) => (
-          <Table.Cell key={index}>{value?.name ?? "N/A"}</Table.Cell>
+        Cell: ({ cell: { value } }) => (
+          `${value?.name ?? "N/A"}`
         ),
       },
       {
         Header: () => <div className="text-right">Total</div>,
         accessor: "total",
-        Cell: ({ row, cell: { value }, index }) => (
-          <Table.Cell key={index}>
-            <div className="text-right">
-              {formatAmountWithSymbol({
-                amount: value,
-                currency: row.original.currency_code,
-                digits: 2,
-              })}
-            </div>
-          </Table.Cell>
+        Cell: ({ row, cell: { value } }) => (
+          <div className="text-right">
+            {formatAmountWithSymbol({
+              amount: value,
+              currency: row.original.currency_code,
+              digits: 2,
+            })}
+          </div>
         ),
       },
       {
         Header: "",
         accessor: "currency_code",
-        Cell: ({ cell: { value }, index }) => (
-          <Table.Cell key={index} className="w-[5%]">
-            <div className="text-right text-grey-40">{value.toUpperCase()}</div>
-          </Table.Cell>
+        Cell: ({ cell: { value } }) => (
+          <span className="w-[5%]">
+            <div className="text-right text-grey-40">{value.toUpperCase()}</div>  
+          </span>
         ),
       },
       {
         Header: "",
         accessor: "country_code",
-        Cell: ({ row, index }) => (
-          <Table.Cell className="w-[5%] pr-2" key={index}>
+        Cell: ({ row }) => (
+          <span className="w-[5%] pr-2">
```

Comment:
Using a span here will not render the same ways as previously did, replace the entire cell content with this:

```suggestion
          <div className="flex rounded-rounded w-full justify-end pr-2">
            <Tooltip
              content={
                isoAlpha2Countries[
                  row.original.shipping_address?.country_code?.toUpperCase()
                ] || row.original.shipping_address?.country_code?.toUpperCase()
              }
            >
              <ReactCountryFlag
                className={"rounded"}
                svg
                countryCode={row.original.shipping_address?.country_code}
              />
            </Tooltip>
          </div>
```